app_data/communities.yaml

```diff
@@ -0,0 +1,98 @@
+  logo: astrothesis.png
+  metadata:
+    title: Astronomy Thesis Collection
+    description: "This community, curated by astronomy librarians, aims to bring together astronomy and astrophysics theses and dissertations. Share the full-text of your cumulative, graduate or PhD level, research with the world by uploading it as open-access to this subject based community. Once added to the community, it can be indexed by ADS, searched for on Zenodo, and browsed to in the community page. The ADS indexes this community once a month."
+    curation_policy: "This collection includes theses and dissertations written to complete Masters or PhD requirements on topics in the fields of astronomy and astrophysics. Other publication types and topics will not be accepted; however, documents that supplement your thesis, such as data sets or videos, are welcome. Although Zenodo accepts all file formats, PDF is strongly preferred for publication submissions. Theses and dissertations written in any language will be accepted, but the metadata record should be created in English. If the NASA ADS has already indexed your thesis, provide us the Bibcode in the Alternate Identifier field before submitting the document."
+    page: "This thematic, community-wide repository curated by astronomy librarians and indexed by the NASA ADS collocates theses and dissertations on astronomy and astrophysics. Even if the ADS already lists your dissertation, you can provide open access to a free, full-text, online version of your own work. Browse the collection on https://zenodo.org/collection/user-astrothesis. Upload your thesis on https://zenodo.org/deposit/?c=astrothesis. For detailed instructions on contributing see:our short video walk-through https://youtu.be/g1fHHJoyp3M or our written walk-through http://wolba.ch/gazette/walk-through-contribute-to-the-astronomy-thesis-collection. Points to Remember! Include your ADS bibcode as "is alternate identifier" in the Related/alternate identifies tab near the bottom. Upload a machine readable PDF (use a tool like Acrobat to OCR if necessary). Know and enter your DOI (Digital Object Identifier). Your institution may have issued one. Enter your DOI in the space provided. Publication date is the year your thesis was accepted. Your Affiliation is your awarding university. The Description field is for your abstract. Use the Thesis tab to list your awarding university and to give credit to your supervisor. For Keywords, use the http://astrothesaurus.org/thesaurus/alphabetical-browse Unified Astronomy Thesaurus. The default http://creativecommons.org/choose. Creative Commons Attribution license allows the ADS to use the full-text of your thesis for searching. Enter the References from your Works Cited section into the references tab individually. This is a nice, but optional step. Include related material (e.g. datasets), by providing their ID and an appropriate description on a new line within Related/alternate identifiers tab."
```

Comment:
minor: I think this one still has an issue because there is a `"` character in the text itself (search for `"is alternate identifier"`). These can be escaped with backslashes (i.e. `\"`) or you can pick another one of the string formats for YAML.